app/src/main/java/com/example/myapplication/csv_to_json.py

```diff
@@ -0,0 +1,35 @@
+import csv, json, datetime
+
+# Specify the directory path
+csv_file_name = r'C:\Users\camil\Robotics\JoinedData\PNCMP2024-MatchExport-Q100.csv' #TODO:check if file name need changing later
+json_file_name = r'C:\Users\camil\Robotics\JoinedData\DCMPQualMatches.json'
+
+all_matches = []
+total_time = None
+current_time = datetime.datetime.hour*1000 + datetime.datetime.minute*100 + datetime.datetime.second + float(1/datetime.datetime.microsecond)
```

Comment:
Make a function that we can call for instead of typing it over and over again

Reply:
Nice spot, this is a pretty common pitfall that newer coders fall into